client/request_types.go

```diff
@@ -118,3 +118,50 @@ type BroadcastTxCommitReply struct {
 	TxHash keys.Address                   `json:"txHash"`
 	Result ctypes.ResultBroadcastTxCommit `json:"result"`
 }
+
+/*
+	ONS Request Types
+*/
+type OnsCreateRequest struct {
+	Owner   []byte        `json:"owner"`
```

Comment:
When dealing with addresses use the `keys.Address` type so it marshals into the form `0x...` in JSON

Reply:
Otherwise it gets encoded as base64 in the JSON